lib/rspec/core/example_group.rb

```diff
@@ -511,10 +506,7 @@ def self.run_after_context_hooks(example_group_instance)
 
       # Runs all the examples in this group.
       def self.run(reporter=RSpec::Core::NullReporter)
-        if RSpec.world.wants_to_quit
-          RSpec.world.clear_remaining_example_groups if top_level?
-          return
```

Comment:
Historically this is for fast-fail right? Which still works so seems like a hangover from previous times...